### ISSUE TITLE
Accessibility issues with <p> lists

### ISSUE THREAD

**Issue by MissArray** [fac-14/Brickworks-Tollington_Project#111]
The Top 10 Tips ordered list has been replaced by simple <p> tags. These cannot be scanned as list items by screen readers. Please restore the ordered list for better accessibility.

https://accessibility.umn.edu/core-skills/bulleted-numbered-lists
